src/router/index.ts

```diff
@@ -1,26 +1,39 @@
-import { createRouter, createWebHistory } from 'vue-router'
-import AuthLayout from '@/layouts/AuthLayout.vue'
+import { createRouter, createWebHistory } from 'vue-router';
+import AuthorizationLayout from '@/layouts/AuthorizationLayout.vue';
+import DefaultLayout from '@/layouts/DefaultLayout.vue';
 
 const router = createRouter({
   history: createWebHistory(),
   routes: [
     {
-      path: '/',
-      component: AuthLayout,
+      path: '/authorization',
+      component: AuthorizationLayout,
+      redirect: { name: 'TheLogin' },
       children: [
         {
           path: 'registration',
-          name: 'registration',
-          component: () => import('@/components/RegistrationForm.vue'),
+          name: 'TheRegistration',
```

Comment:
```suggestion
          name: 'ROUTE_NAMES.RegistrationPage',
```

На конце всех страниц добавить слово `Page`. В константах тоже переименуй.